Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim-buster
+
+# Set the working directory in the container to /app
+WORKDIR /app
+
+# Install git
+RUN apt-get update && \
+    apt-get upgrade -y
```

Comment:
**Suggestion:** Consider cleaning up the apt-get cache after installing packages to reduce the Docker image size.
```suggestion
RUN apt-get update && \
    apt-get upgrade -y && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```